programs/x509/cert.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Looks like an accidental add, as well as `cert.req` and `cert_write_demo.sh` in the same directory.

Reply:
Removed